test/integration/programs/sidebar/action-sidebar.js

```diff
@@ -441,61 +457,92 @@ context('program action sidebar', function() {
 
     cy
       .get('.sidebar')
-      .find('[data-published-region]')
-      .contains('Published')
+      .find('[data-published-region] button')
+      .contains('Off')
       .click();
 
     cy
-      .get('.picklist')
-      .contains('Draft')
+      .wait('@routePatchAction')
+      .its('request.body')
+      .should(({ data }) => {
+        expect(data.attributes.published_at).to.not.be.null;
+      });
+
+    cy
+      .get('.sidebar')
+      .find('[data-published-region] button')
+      .contains('On')
       .click();
 
     cy
       .wait('@routePatchAction')
       .its('request.body')
       .should(({ data }) => {
-        expect(data.attributes.published).to.be.false;
-        expect(data.attributes.behavior).to.equal('standard');
+        expect(data.attributes.published_at).to.be.null;
       });
 
     cy
       .get('.sidebar')
-      .find('[data-published-region]')
-      .contains('Draft')
+      .find('[data-archived-region] button')
+      .contains('Off')
       .click();
 
     cy
-      .get('.picklist')
-      .contains('Conditional')
+      .wait('@routePatchAction')
+      .its('request.body')
+      .should(({ data }) => {
+        expect(data.attributes.archived_at).to.not.be.null;
```

Comment:
Not sure how we want to test this.

This is when the `Published`/`Archived` toggle is switched to `On`. Which means we generate a timestamp (`dayjs.utc().format()`) and send that in the `PATCH` request.

I'm currently just testing if this value is `null` or not.

Do we need to test the exact timestamp? Is that possible to do without creating flake?

Reply:
not null is ok to me.  we could test this with `visitOnClock` by setting the now and expecting that now to be this timestamp